src/main/java/com/rocketden/main/config/WebSocketConfig.java

```diff
@@ -27,6 +27,6 @@ public void configureMessageBroker(MessageBrokerRegistry config) {
 
     @Override
     public void registerStompEndpoints(StompEndpointRegistry registry) {
-        registry.addEndpoint(BaseRestController.BASE_SOCKET_URL + "/join-room-endpoint").withSockJS();
+        registry.addEndpoint(BaseRestController.BASE_SOCKET_URL + "/join-room-endpoint").setAllowedOrigins("*").withSockJS();
```

Comment:
Is there any concern with allowing any origins? I'm ok with this but figured it'd be good to double-check.

Reply:
[This link](https://stackoverflow.com/questions/12001269/what-are-the-security-risks-of-setting-access-control-allow-origin) seems to provide a good overview - thoughts? Seems like at some point if we added proper login/credential stuff it may be worth disabling (at least to be safe), but for now this comment seems reassuring: 

> Imagine what a user would see if they accessed the resource in an incognito window. If you're happy with everyone seeing this content (including the source code the browser received), it's safe to add Access-Control-Allow-Origin: *.

Reply:
That's a good source, thanks for including that - yeah I'm absolutely ok with keeping this for now, and potentially re-evaluating down the line 👍